asreview/webapp/api/projects.py

```diff
@@ -1166,6 +1204,9 @@ def api_export_dataset(project):
         # read the dataset into a ASReview data object
         as_data = read_data(project)
 
+        # Convert -1 to NaN
+        as_data.df["debug_label"] = as_data.df["debug_label"].replace(-1, np.nan)
```

Comment:
```suggestion
        as_data.df["debug_label"] = as_data.df["debug_label"].replace(LABEL_NA, np.nan)
```

Reply:
You can also use None instead of np.nan. This might resolve your issue with unintended casting to floats.

Reply:
Using None is indeed better!